mdl/v1/block.go

```diff
@@ -0,0 +1,32 @@
+package mdl
+
+import (
+	"time"
+)
+
+type Block struct {
```

Comment:
rename to BlockSummary.. it isn't a `Block` by any stretch of the imagination.  a `Block` from a top-level package should represent the best we know of a block.. but that would be in `eos-go` anyway.